src/app/components/Account/ShowMoreTokensLink.tsx

```diff
@@ -45,7 +40,7 @@ export const ShowMoreTokensLink: FC<ShowMoreTokensLinkProps> = ({ account, token
   const pillsSymbols = new Set(pills.map(({ token_contract_addr }) => token_contract_addr))
   const showMoreItems = tokens.filter(({ token_contract_addr }) => !pillsSymbols.has(token_contract_addr))
   const hasERC20 = showMoreItems.some(item => item.token_type === EvmTokenType.ERC20)
-  const targetShowMoreLink = hasERC20 ? erc20link : erc721Link
+  const targetShowMoreLink = hasERC20 ? erc20link : '???' // Shouldn't happen, since we only have ERC20 tokens.
```

Comment:
I assume it's temp value. showMoreItems can only open erc20 since now.

Reply:
Right. Simplified.